src/components/BackgroundText.vue

```diff
@@ -26,7 +26,7 @@ export default {
   left: 20px;
   color: white;
   line-height: 1;
-  z-index: 3;
+  z-index: 1;
```

Comment:
I'm not really comfortable with all the z-index changes that keep happening, but that's not an issue with your pull request. More a problem with the initial setup of this project.

Reply:
A lot of it was flattening and consolidation. At first, I just deleted nearly all of them, then tracked down what broke and replaced it with as low a value as possible where needed.